src/BridgeMaker.js

```diff
@@ -7,7 +7,11 @@ const BridgeMaker = {
    * @param {function(): number} generateRandomNumber 무작위 값을 생성해주는 함수
    * @return {string[]} 입력받은 길이에 해당하는 다리 모양. 위 칸이면 U, 아래 칸이면 D로 표현해야 한다.
    */
-  makeBridge(size, generateRandomNumber) {},
+  makeBridge(size, generateRandomNumber) {
+    return Array.from({ length: size }, () =>
+      generateRandomNumber()
+    ).map((command) => (command === 1 ? 'U' : 'D'));
```

Comment:
1은 true로 취급되니 해당부분은 생략 가능할 것 같습니다! 
```suggestion
    ).map((command) => (command ? 'U' : 'D'));
```
아니면 코드리뷰를 하다가 알게되었는데 숫자와 방향을 NUMBER_TO_DIRECTION으로 상수화하고 
`Array.from({ length: size }, () => { NUMBER_TO_DIRECTION[generateRandomNumber()] })`
하는 방식도 좋은 것 같습니다!